### PR TITLE
fix(cli): improve error messages

### PR DESCRIPTION
**Pull request by jfeodor**
* Do not print `delete` errors twice
* Print specific error messages for common cases
* Refactor error printing and move printing to `cmd` package

### CODE REVIEW

File: cmd/output/error.go [hunk 1]
```diff
@@ -76,3 +80,54 @@ func PrintErrorInvalidOrganizationSlug(slug string) {
 func PrintErrorInvalidAppSlug(appSlug string) {
 	PrintError("Invalid app %q.", "Must contain only lower-case alphanumerical characters and dashes.", appSlug)
 }
+
+func PrintErrorAppNotFound(ai appident.AppIdentifier) {
+	PrintError(
+		"App not found",
+		"The app you are trying to delete \"%s/%s\" cannot be found. Did you specify the correct organization and app slug?",
```
Comment (moroderNumerous) [inline cmd/output/error.go:87]: The naming of the function makes it seem like it was intended to be generic and used multiple places 😄 
So we should replace the word "delete" with something generic like "access"/"manage"  or just  remove "you are trying to delete"

File: cmd/output/error.go [hunk 2]
```diff
@@ -76,3 +80,54 @@ func PrintErrorInvalidOrganizationSlug(slug string) {
...
+}
+
+func PrintAppError(err error, ai appident.AppIdentifier) {
+	switch {
+	case errors.Is(err, app.ErrAccesDenied):
+		PrintErrorAccessDenied(ai)
+	case errors.Is(err, app.ErrAppNotFound):
+		PrintErrorAppNotFound(ai)
+	default:
+		PrintErrorDetails("Error occurred deleting app.", err)
```
Comment (moroderNumerous) [inline cmd/output/error.go:108]: This also seems too specific for the naming function 😄

**Review by moroderNumerous — COMMENTED** [review]
Great clean up 😄  Also seems like responsibility of where a error should be printed to the user is much clearer with these fixes!

File: cmd/output/error.go (hunk 2 above)
Comment (jfeodor) [reply cmd/output/error.go:108]: Not sure what you mean?

Comment (jfeodor) [reply cmd/output/error.go:108]: Ah, good point.

**Review by moroderNumerous — APPROVED** [review]

**Pull request merged**